std_msgs/UInt64     id
std_msgs/Float64    prob
std_msgs/Float64[3] mean
std_msgs/Float64[9] covariance
std_msgs/Float64[9] eigen_vectors
std_msgs/Float64[3] eigen_values
